### PR TITLE
Add infos about enable logging over UDP in rsyslog

### PR DESCRIPTION
**Pull request by big-r81**
## Overview

Add configuration infos for rsyslog to enable logging to facility

## GitHub issue number

https://github.com/apache/couchdb/issues/3882

### CODE REVIEW

File: src/config/logging.rst [hunk 1]
```diff
@@ -137,3 +137,13 @@ Logging options
 
             [log]
             syslog_facility = local2
+
+    .. note::
+        CouchDB's ``syslog`` only knows how to use UDP logging. Please ensure that your
+        ``syslog`` server has UDP logging enabled.
```
Comment (nickva) [inline src/config/logging.rst:143]: Well written. Let's add a note that `syslog_host` must be specified for the logging to work.  `local2` is already a default, so technically not needed to specified.

**Review by nickva — APPROVED** [review]
+1, but see a note about possibly adding that syslog_host is mandatory for syslog to work

File: src/config/logging.rst (hunk 1 above)
Comment (big-r81) [reply src/config/logging.rst:143]: 50% of these two sentence did you contribute ;-)
The `local2` example was in the docs before. Should we keep it or use another value?

**Comment by big-r81** [comment]
Added a note for `syslog_host`

**Pull request merged**